rust-src/concordium_base/src/curve_arithmetic/secret_value.rs

```diff
@@ -117,7 +120,17 @@ mod tests {
         };
     }
 
-    macro_test_value_to_byte_conversion!(value_to_byte_conversion_bls12_381_g1_affine, G1Affine);
-
-    macro_test_value_to_byte_conversion!(value_to_byte_conversion_bls12_381_g2_affine, G2Affine);
+    // TODO: the code used to be diefined using the affine representation, but
+    // ArkWorks' BLS does not implement `CurveGroup` for the affine representation.
+    // Is it important that it's the affine representation? For now, the BLS
+    // projective representation will be used here.
+    macro_test_value_to_byte_conversion!(
+        value_to_byte_conversion_bls12_381_g1_affine,
+        ArkGroup<G1Projective>
+    );
```

Comment:
Just rename it to use projective. This is just a test, it is not important that it was for affine, since we don't have the affine groups anymore.

Reply:
Done.